pkg/charts/dependencies.go

```diff
@@ -257,16 +257,21 @@ func UpdateHelmMetadataWithDependencies(fs billy.Filesystem, mainHelmChartPath s
 		for _, d := range chart.Metadata.Dependencies {
 			if d.Name == dependencyName {
 				d.Repository = fmt.Sprintf("file://./charts/%s", dependencyName)
-				d.Version = "" // Local chart archives don't need a version
```

Comment:
d.Version needs a version here as well just like in line 268. I checked d.Version is sometimes empty.